### ISSUE TITLE
Bug: vanilla interpreter error

### ISSUE THREAD

**Issue by redsun82** [facebook/starlark-rust#133]
When running the vanilla `starlark_bin` interpreter, I get:
```
Error: `starlark:False.bzl` does not have an absolute path component
```
The error seems to stem from [this piece of code](https://github.com/facebook/starlark-rust/blob/42251320691874b139e07d60be37218ca1b1c761/starlark_bin/bin/eval.rs#L126-L131). I was able to get past it by replacing that with
```rust
        let uri = Url::parse(&format!("starlark:/native/builtin.bzl"))?;
        let uri = LspUrl::try_from(uri)?;
        for (name, item) in globals.documentation().members {
            builtin_docs.insert(uri.clone(), item.render_as_code(&name));
            builtin_symbols.insert(name, uri.clone());
        }
```
though I have no idea whether that is the correct thing to do.

I know vanilla `starlark_bin` isn't really supposed to be used for a domain specific implementation, but it's still a nice entry point when trying the project the first time.

**Comment by ndmitchell** [comment]
What code are you running? Is this for the IDE?

**Comment by redsun82** [comment]
I was just running the binary with no arguments, which should work as a vanilla interpreter (just `cargo run`). This was mostly intended for experimenting with the language. That said, while I can confirm this was happening on the commit I tried it with (https://github.com/facebook/starlark-rust/commit/9a7b00eecd723c026fb8dc4ffdc9a5729c6ce384), I can also confirm it's no longer happening on `main`. It was fixed by https://github.com/facebook/starlark-rust/commit/54d7cfd299c8a2c67afd2ef6ec4da49ce5c2c494 👍
